### ISSUE TITLE
Only Movies

### ISSUE THREAD

**Issue by SirdLay** [Fallenbagel/jellyseerr#698]
### Description

Hey is there a way to list only movies section no series.

### Desired Behavior

For all users, they can view and request only movies.

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

**Comment by Fallenbagel** [comment]
You can turn off request permissions for shows for all users. You can also edit the sliders on discover page and remove the series ones and leave the movies one only.

**Comment by SirdLay** [comment]
@Fallenbagel  yup checked that one. It's good but it would be better to have an option to disable it f from left pane and Trending cause even if we disable series in discover it's still accessible from left pane.

**Comment by Fallenbagel** [comment]
> @Fallenbagel  yup checked that one. It's good but it would be better to have an option to disable it f from left pane and Trending cause even if we disable series in discover it's still accessible from left pane.

Yeah from the panel it is. But they can't request but yeah I get what you're saying c:

**Comment by SirdLay** [comment]
@Fallenbagel  no rush, if you ever find some spare time. btw thanks for the great effort

**Comment by gauthier-th** [comment]
Partial duplicate of #163
